modules/display.c

```diff
@@ -1204,6 +1209,31 @@ xlat(int src_lo, int src_hi, int dst_lo, int dst_hi, int val)
  * DATAPIPE_TRACKING
  * ========================================================================= */
 
+/** PackageKit Locked property is set to true during sw updates */
```

Comment:
Looks like you copy pasted this from somewhere else and forgot to update the comment :) (same thing for below comment)

Reply:
:laughing: :laughing: I feel so stupid right now.